rdkit/ML/UnitTestAnalyzeComposite.py

```diff
@@ -52,7 +52,7 @@ def test1_Issue163(self):
 
     self.assertEqual(res[0][0],'BALABANJ')
     self.assertEqual(res[1][0],'BERTZCT')
-    self.assertEqual(res[-1][0],'FR_UNBRCH_ALKANE')
+    self.assertEqual(res[-1][0],'VSA_ESTATE9')
```

Comment:
Interesting that we see a change here now. Is better to do the sort in the test.